jigsawpy/libsaw.py

```diff
@@ -4,6 +4,7 @@
 import numpy as np
 import inspect
 import platform
+import sys
 
 from pathlib import Path
 
```

Comment:
To be honest, lines 67 through 71 are actually doing what they're "supposed" to do.

Reply:
@jreniel so what's going on here is support for the various ways that `jigsaw`'s backend `c++` layer can be installed: 

1. If `jigsaw` is installed "standalone" via `jigsawpy` (via `python setup.py build_external; python setup.py install` etc) the binaries should be installed with `jigsawpy` in `../jigsawpy/_bin` and '../jigsawpy/_lib'. The first block of `JLIBNAME` statements look for `libjigsaw` locally in this case.

2. If `jigsaw` is installed separately (via its `conda` package for example) the second block of `JLIBNAME` statements look for the binaries globally on the system paths (`usr/local/bin/` and `usr/local/lib`, etc on `linux`). In this case, a single installation of `jigsaw`'s backend may be shared by multiple frontend layers --- a single backend could be shared by both the `Python` and `MATLAB` interfaces for example.

Let me know if this isn't working for you.

Reply:
Thanks for the clarification, I will try with option #1.